db/sqlglue.c

```diff
@@ -5562,7 +5575,7 @@ int sqlite3BtreeMovetoUnpacked(BtCursor *pCur, /* The cursor to be moved */
         /* filter the supported operations */
         if (bias != OP_SeekLT && bias != OP_SeekLE && bias != OP_SeekGE &&
             bias != OP_SeekGT && bias != OP_NotExists && bias != OP_Found &&
```

Comment:
@dorinhogea: This is the second of two places that needed to change to get the "write_remsql_names" test to pass on this branch.  Please review for correctness.

Reply:
Change is correct. See #2413.